cmake/AddInputFileTests.cmake

```diff
@@ -1,6 +1,8 @@
 # Distributed under the MIT License.
 # See LICENSE.txt for details.
 
+find_package(PythonInterp REQUIRED)
```

Comment:
Would you mind either documenting how to write these tests in the Dev Guide (maybe change `Writing Unit Tests` to `Writing Tests`), or filing an issue to do that?

Reply:
Opened #1342 (don't want to hold this PR up with discussions on documentation, which tend to be length)